src/components/media_manager/include/media_manager/audio/a2dp_source_player_adapter.h

```diff
@@ -37,11 +37,17 @@
 #include "protocol/common.h"
 #include "media_manager/media_adapter_impl.h"
 #include "utils/threads/thread_delegate.h"
+#include "connection_handler/connection_handler_impl.h"
 
 namespace protocol_handler {
 class SessionObserver;
 }
 
+
```

Comment:
It looks like the style checking script has an issue with this newline, I'll remove it via GitHub